app/components/workflow-files/index.js

```diff
@@ -12,6 +13,13 @@ export default class WorkflowFiles extends Component {
   @service currentUser;
   @service flashMessages;
 
+  @tracked doi = null;
+
+  constructor() {
+    super(...arguments);
+    this.doi = this.args.doi;
```

Comment:
I'm surprised that you needed the constructor here (as opposed to just setting the field above), but if it works I think it's fine to leave as is.